docker/hq-compose.yml

```diff
@@ -81,20 +81,25 @@ services:
       - ${VOLUME_PREFIX}${BLANK_IF_TESTS-elasticsearch:}/usr/share/elasticsearch/data
 
   zookeeper:
-    image: wurstmeister/zookeeper
+    image: wurstmeister/zookeeper:3.4.6
     ports:
       - "2181:2181"
     volumes:
-      - ${VOLUME_PREFIX}${BLANK_IF_TESTS-zookeeper:}/opt/zookeeper-3.4.13/data
+      - ${VOLUME_PREFIX}${BLANK_IF_TESTS-zookeeper:}/opt/zookeeper-3.4.6/data
```

Comment:
does kafka not work with 3.4.13?

Reply:
Did many changes then i do not recall what was the result , now that its passing  let try the latest versions of Zookeeper and Kafka

Reply:
wurstmeister does have the latest version of Zookeeper , used a different version from dockerhub it passes now   https://travis-ci.org/github/dimagi/commcare-hq/builds/669199056